src/routes/root.tsx

```diff
@@ -45,13 +47,13 @@ export async function loader() {
 export async function action({ request }: ActionFunctionArgs) {
   const formData = await request.formData();
   const actionType = formData.get("action-type");
-  if (actionType === "add-feed") {
-    const req = await createRequest("contentFeed");
+  const itemType = formData.get("item-type");
+  if (itemType === "request") {
```

Comment:
these conditions are not needed it should be explicit instead and handle inside the create requests method